src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Lower.java

```diff
@@ -4440,6 +4440,65 @@ public void visitTry(JCTry tree) {
         super.visitTry(tree);
     }
 
+    @Override
```

Comment:
As usual, I suggest to add some brief comment with the shape of the generated code.

Reply:
Done here:
https://github.com/openjdk/jdk/pull/18509/commits/146513580f96abd5feb7886bd1191805cc93403b#diff-bc0df6dce7f74078bfca1e90bec75d7bb3d8b338933be725da78f0a8a7a0c78dR4445

Please let me know if that needs some tweaks.

Thanks for the comment!